### ISSUE TITLE
Assignment Lecture 4

### ISSUE THREAD

**Issue by pepsi1234** [goldshakil/ElectricalProgrammingCourse2020#55]
half of lecture 04-05 pdf is same with lecture 3 pdf. so all codes in lecture 3 pdf are in lecture 04-05.
should i copy all codes that i copied for assignment 3 too? or just copy codes in lecture 04-05 except for codes i copied past?

**Comment by sunilsrivatsav** [comment]
Yes you can copy the new codes from lecture 04-05 and skip the codes which are repeating.

**Comment by pepsi1234** [comment]
> Yes you can copy the new codes from lecture 04-05 and skip the codes which are repeating.

thanks to answer my question :)
